app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -2152,6 +2172,22 @@ public void setPointerMode(@PointerMode int mode) {
         queueRunnable(() -> setPointerModeNative(mode));
     }
 
+    @Override
+    public void setHandTrackingEnabled(boolean value) {
+        mIsHandTrackingEnabled = value;
+        queueRunnable(() -> setHandTrackingEnabledNative(value));
+    }
+
+    @Override
+    public boolean isHandTrackingEnabled() {
+        return mIsHandTrackingEnabled;
+    }
+
+    @Override
+    public boolean isHandTrackingSupported() {
+        return mIsHandTrackingEnabled;
```

Comment:
Why are we returning the same attribute than above?

Reply:
A mistake.